packages/loader/execution-context-loader/src/webWorkerLoader.ts

```diff
@@ -71,12 +70,4 @@ export class WebWorkerLoader implements ILoader, IFluidRunnable, IFluidRouter {
     public async resolve(request: IRequest): Promise<IContainer> {
         return this.proxy.resolve(request);
     }
-
-    public async createDetachedContainer(source: IFluidCodeDetails): Promise<IContainer> {
```

Comment:
i'll let you decided if its useful or not, but we could have an IHostLoader interface, or somthing similar if an interace has any utility

Reply:
Was on the fence about actually doing that when working on the change but I think it makes sense.  It simplifies the test changes (which I assume were written that way in case of other Loaders) and interfaces are cheap